src/build_scripts/buildlib.cprnc

```diff
@@ -52,14 +54,26 @@ def buildlib(bldroot, installpath, case):
 
     cmake_args = get_standard_cmake_args(case, "ignore_sharedpath", shared_lib=True)
     os.environ["CIMEROOT"] = cimeroot
-    cmake_cmd = ". ./.env_mach_specific.sh && NETCDF=$(dirname $(dirname $(which nf-config))) cmake {cmake_args} -DMPILIB=mpi-serial -DDEBUG=FALSE -C Macros.cmake {cimeroot}/tools/cprnc -DCMAKE_PREFIX_PATH={dest_path} -DBLDROOT={bldroot}".\
+    cmake_cmd = ". .env_mach_specific.sh && NETCDF=$(dirname $(dirname $(which nf-config))) cmake {cmake_args} -DMPILIB=mpi-serial -DDEBUG=FALSE -C Macros.cmake {cimeroot}/tools/cprnc -DCMAKE_PREFIX_PATH={dest_path} -DBLDROOT={bldroot}".\
```

Comment:
Because it was redundant. `./foo` is the same path as `foo`

Reply:
@jgfouca - It looks like this change was backed out, but for future cases like this: It *can* be important to have `./` before a file name that you're sourcing - otherwise, if dot isn't in your path, you can get the wrong results.

Reply:
What Bill said.  It is not always true that ./foo and foo are the same commands.

Reply:
@rljacob , I ended up reverting this change. For what it's worth, the ./ is definitely redundant here; notice the ". " at the start of the command; we are sourcing the env_mach_specific.sh file before running cmake. Source takes a path, but does not scan PATH, so "./foo" and "foo" are the same thing in this context.

Reply:
I just googled it and of course I'm dead wrong. Source does scan PATH... I learned something new today.

Reply:
Yup, I was bitten by that a few years ago: I struggled to understand why it wasn't working right to say `source .env_mach_specific.sh`, and after a lot of head-scratching found out that it was sourcing a version of the file from the directory containing the `cprnc` build on cheyenne, which I had in my path, rather than the one in my current directory!

Reply:
It's very counterintuitive since the script being sourced is an argument to the source command and not the command itself.

Reply:
I agree.